references/cohort_consensus.fasta

```diff
@@ -0,0 +1 @@
+NC_045512.2.fasta
```

Comment:
I might be overlooking something, but what is this for?

Reply:
I think this is an older commit because the pull request is from a branch that didn't get `pull --rebase` or `merge` from the latest sars-cov2-dev.

@kpj Could you fix that ?

Reply:
Done!